### DISCUSSION TITLE
SMPlayer suddenly does not work

### DISCUSSION THREAD

**Discussion by StanislavBriza** [smplayer-dev/smplayer#578] [Q&A] [answered]
hello everyone, I was supper happy with SMPlayer until recently. A few days ago evertime I start smplayer, it just flashes as a small window is the corner. I don´t see any actual program menu or window. I can´t do anything or get into settings. I have win11 64bit, updated, I restarted pc, I unnistalled the program and installed again. Didn´t help. I tried older version, again no success. Does anybody have any clue what is going on? I didn´t change any setting and rest of pc behaves kinda normal. Thanks for tips

**Comment by smplayer-dev** [comment, marked as answer]
Open C:\Users\your_user\AppData\Local\Temp and delete the file qtsingleapp-smplay-*lockfile if it exits.

**Reply by StanislavBriza** [reply]
thanks Ricardo, worked like a charm!
